solr/server/etc/security.policy

```diff
@@ -228,4 +228,7 @@ grant {
 
   // expanded to a wildcard if set, allows all networking everywhere
   permission java.net.SocketPermission "${solr.internal.network.permission}", "accept,listen,connect,resolve";
+
+  // Used by SolrProcessManager to find commandLine of a running Windows process
+  permission java.io.FilePermission "C:\\Windows\\*\\wmic.exe", "execute";
```

Comment:
This is not yet tested..

Reply:
Thinking about it, since status tool is only used from CLI and not from Solr server, we don't need this permission, since CLI runs without security mgr. I'll remove this.